### PR TITLE
feat: add cantonese translations PE-1462

### PR DESCRIPTION
**Pull request by jdaev**
Adds arb file for Cantonese.

### CODE REVIEW

**Review by karlprieb — DISMISSED** [review]

**Review by karlprieb — APPROVED** [review]

File: lib/main.dart [hunk 1]
```diff
@@ -81,7 +83,11 @@ class _AppState extends State<App> {
             supportedLocales: const [
               Locale('en', ''), // English, no country code
               Locale('es', ''), // Spanish, no country code
-              Locale('zh', ''), // Chinese (Mandarin), no country code
+              Locale.fromSubtags(languageCode: 'zh'), // generic Chinese 'zh'
+              Locale.fromSubtags(
+                languageCode: 'zh',
+                countryCode: 'HK',
+              ), // generic traditional Chinese 'zh_Hant'
```
Comment (matibat) [inline lib/main.dart:90]:
```suggestion
              Locale.fromSubtags(languageCode: 'zh'), // Simplified Chinese, no country code
              Locale.fromSubtags(
                languageCode: 'zh',
                countryCode: 'HK',
              ), // Chinese, Hong Kong
```

Comment (matibat) [reply lib/main.dart:90]: BTW is that equivalent to 
```suggestion
              Locale('zh', ''), // Chinese (Mandarin), no country code
              Locale('zh', 'HK'), // Chinese (Cantonese), Hong Kong
```

**Review by matibat — APPROVED** [review]
Now it is working fine! Thanks! 🚀 

_Left a non-blocking comment there_

**Comment by matibat** [comment]
Nit: there's a tiny conflict

**Pull request merged**